text/0000-use-changelog.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2016-07-13
+- RFC PR: (leave this empty)
+- Nix Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+We use a change log file to document API changes of released versions relative
+to their predecessors.
+
+# Motivation
+[motivation]: #motivation
+
+Change logs are a common and convenient way to document API changes in a single
+location. Being able to easily find notes on breaking changes is particularly
+useful.
+
+# Detailed design
+[design]: #detailed-design
+
+We follow the conventions laid out in [Keep A CHANGELOG](https://github.com/olivierlacan/keep-a-changelog/tree/18adb5f5be7a898d046f6a4acb93e39dcf40c4ad).
+
```

Comment:
Might be useful to include the following:

* Guidlines for contributors related to updating the changelog (probably encourage it but make it a soft requirement)
* Note responsibility for maintainer to update CHANGELOG before release including creating a new unreleased section.